src/main/resources/processes/MPAM_TRIAGE_ON_HOLD.bpmn

```diff
@@ -36,79 +38,209 @@
+      <bpmn:incoming>SequenceFlow_194yqh0</bpmn:incoming>
+      <bpmn:outgoing>SequenceFlow_1jxsbqx</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:serviceTask id="ServiceTask_1smdf47" name="Clear CampaignType" camunda:expression="${bpmnService.blankCaseValues(execution.getVariable(&#34;CaseUUID&#34;),execution.processBusinessKey,&#34;CampaignType&#34;)}">
+      <bpmn:incoming>SequenceFlow_0gnfg5d</bpmn:incoming>
+      <bpmn:outgoing>SequenceFlow_1arugxn</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:sequenceFlow id="SequenceFlow_0g0fmye" sourceRef="ExclusiveGateway_0kxi73e" targetRef="ServiceTask_18u5rz3">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${valid == false}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="SequenceFlow_1arugxn" sourceRef="ServiceTask_1smdf47" targetRef="ServiceTask_18u5rz3" />
+    <bpmn:sequenceFlow id="SequenceFlow_1juicl4" sourceRef="ServiceTask_18u5rz3" targetRef="UserTask_1ql7p2r" />
+    <bpmn:sequenceFlow id="SequenceFlow_0gnfg5d" name="PutOnCampaign" sourceRef="ExclusiveGateway_0crgphp" targetRef="ServiceTask_1smdf47">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${TriageOnHoldOutcome == "PutOnCampaign"}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="SequenceFlow_1s7fzjp" sourceRef="UserTask_1ql7p2r" targetRef="ExclusiveGateway_1pou7z0" />
+    <bpmn:sequenceFlow id="SequenceFlow_1h096pp" sourceRef="ExclusiveGateway_1pou7z0" targetRef="ExclusiveGateway_0kxi73e">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${DIRECTION == "FORWARD"}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="SequenceFlow_194yqh0" sourceRef="ExclusiveGateway_0kxi73e" targetRef="ServiceTask_0ms9mqv">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${valid == true}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="SequenceFlow_1yex0y8" name="else" sourceRef="ExclusiveGateway_0crgphp" targetRef="EndEvent_1golwf2">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${TriageOnHoldOutcome != "PutOnCampaign"}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="SequenceFlow_1jxsbqx" sourceRef="ServiceTask_0ms9mqv" targetRef="EndEvent_1golwf2" />
+    <bpmn:sequenceFlow id="SequenceFlow_1jktdi9" sourceRef="ExclusiveGateway_1pou7z0" targetRef="ServiceTask_05nlmnl">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${DIRECTION == "BACKWARD"}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
   </bpmn:process>
   <bpmndi:BPMNDiagram id="BPMNDiagram_1">
     <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="MPAM_TRIAGE_ON_HOLD">
       <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_1">
-        <dc:Bounds x="119" y="172" width="36" height="36" />
+        <dc:Bounds x="119" y="483" width="36" height="36" />
```

Comment:
You've moved the boxes around so these lines got modified. In future you can prevent this from happening via `git add --patch`.

Reply:
I needed to move these lines, this change is necessary to make the bmpn readable. Some spacing changes are always done when we add new components.